pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -513,10 +513,17 @@ public void removeNamespaceAntiAffinityGroup(@PathParam("property") String prope
     @ApiOperation(hidden = true, value = "Enable or disable broker side deduplication for all topics in a namespace")
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
             @ApiResponse(code = 404, message = "Property or cluster or namespace doesn't exist") })
-    public void modifyDeduplication(@PathParam("property") String property, @PathParam("cluster") String cluster,
-            @PathParam("namespace") String namespace, boolean enableDeduplication) {
+    public void modifyDeduplication(@Suspended AsyncResponse asyncResponse, @PathParam("property") String property,
+                                    @PathParam("cluster") String cluster, @PathParam("namespace") String namespace,
+                                    boolean enableDeduplication) {
         validateNamespaceName(property, cluster, namespace);
-        internalModifyDeduplication(enableDeduplication);
+        internalModifyDeduplicationAsync(enableDeduplication)
+                .thenAccept(__ -> asyncResponse.resume(Response.ok().build()))
```

Comment:
```suggestion
                .thenAccept(__ -> asyncResponse.resume(Response.noContent().build()))
```

Reply:
From the logs, we can see that before this patch, the response HTTP code is 204.

```
2022-05-16T08:47:05,257 - INFO  - [pulsar-web-30-16:Slf4jRequestLogWriter@62] - 127.0.0.1 - - [16/May/2022:08:47:05 +0800] "POST /admin/v2/namespaces/my-property/my-ns/deduplication HTTP/1.1" 204 0 "-" "Pulsar-Java-v2.11.0-SNAPSHOT" 6
```